pkgs/by-name/pk/pkgsite/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGo123Module,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+
+buildGo123Module rec {
```

Comment:
Did you intentionally choose buildGo123Module for this package? Other Go packages in this PR are using buildGoModule.

Reply:
Yes, it was a while ago since I wrote this derivation, the go version was 1.22 then in nixpkgs and pkgsite required go1.23+ in its go.mod. I will change it now.